bears/generate_package.py

```diff
@@ -113,7 +117,13 @@ def main():
     else:
         bear_version = repr(bear_version) + '.' + str(int(time.time()))
 
-    for bear_file_name in sorted(set(glob('bears/**/*Bear.py'))):
+    if args.bear:
+        bear_files = [f for f in sorted(set(glob('bears/**/*Bear.py')))
+                      for x in args.bear if x in f]
```

Comment:
Sure will add comments explaining them. It is actually a `list comprehension` in python. It is actually reducing use of a `for loop` and `two if statements`